docs/README.md

```diff
@@ -14,22 +14,20 @@ to libvirt domains based on the state of those resources.
  * virt-handler: This is a daemon that runs on each Kubernetes node. It is
    responsible for monitoring the state of VMIs according to Kubernetes and
    ensuring the corresponding libvirt domain is booted or halted accordingly.
- * virt-launcher: This component is a place-holder, one per running VMI. Its
-   job is to remain running as long as the VMI is defined. This simply prevents a
-   crash-loop state.
- * ha-proxy: This daemon proxies connections from 192.168.200.2 to the running
-   master node--making it possible to establish connections in a consistent
-   manner.
+ * virt-launcher: This component is a place-holder, one per running VMI. It
+   contains the running VMI and remains running as long as the VMI is defined.
 
 ### Scripts
 
- * `cluster/sync.sh`: After deploying a fresh vagrant environment, or after
-   making changes to code in this tree, this script will sync the Pods and
-   DaemonSets in the running KubeVirt environment with the state of this tree.
- * `cluster/kubectl.sh`: This is a wrapper around Kubernete's kubectl command so
+ * `cluster/kubectl.sh`: This is a wrapper around Kubernetes' kubectl command so
```

Comment:
You could add `cluster/cli.sh` and `cluster/virtctl.sh` which are mentioned on the `getting-started` guide.